test/common/mocks/event/mocks.cc

```diff
@@ -0,0 +1,23 @@
+#include "mocks.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+using testing::_;
+using testing::Assign;
+using testing::DoAll;
+using testing::Invoke;
+using testing::NiceMock;
+using testing::Return;
+using testing::ReturnNew;
+using testing::ReturnPointee;
+using testing::SaveArg;
+
+namespace Envoy {
+namespace Event {
+
+// MockFileEvent::MockFileEvent() = default;
```

Comment:
?

Reply:
default is default, forgot to delete, thanks!